pygmt/helpers/decorators.py

```diff
@@ -188,9 +186,9 @@
                 - **d**\|\ **D** - define a gap when there is a large enough
                   distance between coordinates (upper case to use projected
                   coordinates).
-                - [*col*]\ **z** - define a gap when there is a large enough
-                  change in the data in column *col* [default *col* is 2 (i.e.,
-                  3rd column)].
+                - **z** - define a gap when there is a large enough change in
+                   the z data. Use **+c**\ *col* to change the z data column
+                   [Default *col* is 2 (i.e., 3rd column)].
```

Comment:
Remove one whitespace here.
```suggestion
                  the z data. Use **+c**\ *col* to change the z data column
                  [Default *col* is 2 (i.e., 3rd column)].
```